metabase/metabase
# Create a new e2e test group: dashboard-filters-2

Target PR: #48289

### Description

Tests in #48289 were timing out due to 45min limit: https://github.com/metabase/metabase/actions/runs/11276760715/job/31361810208?pr=48289

This PR moves new `dashboard-filters-query-stages.cy.spec.ts` suite to a new group.
This suite has been introduced in epic #46519 (not in `master` yet).
It's a separate PR to avoid having a huge diff.

### How to verify

Only `markdown-link-check` fails in CI.
And possibly sharing tests (see https://metaboat.slack.com/archives/C0643FZ5E83/p1728655827106879?thread_ts=1728654843.510599&cid=C0643FZ5E83).
